packages/alchemy/src/react/hooks/useSignMessage.ts

```diff
@@ -0,0 +1,71 @@
+"use client";
```

Comment:
nit jsdocs

Reply:
why do we need "use client" here? it's the responsibility of app developers to set these if they are using nextjs. this line is irrelevant for apps that do not use nextjs?

Reply:
Yeah it's irrelevant out of nextjs, but future proofs for nextjs client side rendering

See moldys pr here https://github.com/alchemyplatform/aa-sdk/pull/539#discussion_r1545463146